domains/vercel.cedrickfeze.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "cedrickdev-github"
```

add a contact method 
```suggestion
        "username": "cedrickdev"
```